config/config.exs

```diff
@@ -22,7 +22,11 @@ config :logger,
     [module: Membrane.SRTP.Encryptor, function: "handle_event/4", level_lower_than: :error]
   ]
 
-config :membrane_telemetry_metrics, enabled: true
+telemetry_enabled = true
+
+config :membrane_telemetry_metrics, enabled: telemetry_enabled
+
+config :membrane_opentelemetry, enabled: telemetry_enabled
```

Comment:
I would consider renaming those packages to `membrane_metircs` and `membrane_traces` respectively. Not in this PR, but in general. We would end up with

```elixir
config :membrane_metrics, enabled: true
config :membrane_traces, enabled: true
```

cc @mat-hek